src/better_lane_follow/CMakeLists.txt

```diff
@@ -0,0 +1,69 @@
+cmake_minimum_required(VERSION 2.8.3)
+project(better_lane_follow)
```

Comment:
We should probably change the name to just `lane-following`, seeing as it looks like we're just going to toss our first attempt